app/controllers/api/v1/deliveries_controller.rb

```diff
@@ -3,6 +3,28 @@ class DeliveriesController < Api::V1::ApiController
 
     load_and_authorize_resource :delivery, parent: false
 
+    resource_description do
+      short 'Get, create, and update delivery.'
+      formats ['json']
+      error 401, "Unauthorized"
+      error 404, "Not Found"
+      error 422, "Validation Error"
+      error 500, "Internal Server Error"
+    end
+
+    def_param_group :delivery do
+      param :delivery,  Hash, required: true do
+        param :offer_id, String, desc: "Id of Offer to which delivery belongs."
+        param :contact_id, String, allow_nil: true, desc: "Id of Offer to which delivery belongs."
+        param :schedule_id, String, allow_nil: true, desc: "Id of Offer to which delivery belongs."
+        param :delivery_type, ["Alternate", "Drop Off", "Gogovan"], desc: "Delivery type."
+        param :start, String, allow_nil: true, desc: "Not yet used"
+        param :finish, String, allow_nil: true, desc: "Not yet used"
+      end
+    end
+
+    api :POST, '/v1/deliveries', "Create a delivery"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.